demos/custom-tabs-headers-demo/src/main/java/org/chromium/customtabsdemos/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+package org.chromium.customtabsdemos;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.browser.customtabs.CustomTabsCallback;
+import androidx.browser.customtabs.CustomTabsClient;
+import androidx.browser.customtabs.CustomTabsIntent;
+import androidx.browser.customtabs.CustomTabsService;
+import androidx.browser.customtabs.CustomTabsServiceConnection;
+import androidx.browser.customtabs.CustomTabsSession;
+import android.content.ComponentName;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.Browser;
+
+public class MainActivity extends AppCompatActivity {
+
+    private CustomTabsSession mSession;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        String url = "https://padr31.github.io/";
```

Comment:
Maybe use this as `private static final string URL = "https://padr31.github.io/"`?